util/regexp-assemble/data/933160.data

```diff
@@ -24,7 +24,7 @@
 
 ##!+i
 ##!^\b
-##!$(?:\s|/\*.*\*/|//.*|#.*)*\(.*\)
+##!$\(.*\)
```

Comment:
After removing whitespace and comment markers you might miss this:
```
=array_map\n#dummy comment\n('gotcha')
```
which would be converted to:
```
=array_mapdummycomment('gotcha')
```
Removing whitespace could be ok I guess but removing comment markers will break the rule.

Reply:
yep, your example payload doesn't match anymore :/ The problem we need to solve is just the `//.*` part of the regex. I'm switching this to draft while looking for a solution tnx

Reply:
Maybe you can work around it by matching the end of the line, since `//` will comment everything up to the end:
```
//[^\n\r]*[\n\r]\(.*\)
```